tests/integrations/server/mod.rs

```diff
@@ -25,7 +24,7 @@ use kvproto::tikvpb::{
 macro_rules! unary_call {
     ($name:tt, $req_name:tt, $resp_name:tt) => {
         fn $name(&mut self, ctx: RpcContext<'_>, _: $req_name, sink: UnarySink<$resp_name>) {
```

Comment:
grpcio 0.9 provides the default implementation for the methods, these macros can be removed now.

Reply:
There was a Mocked Raft Client. any suggestion on how to rewrite that part?

Reply:
https://github.com/tikv/tikv/blob/ec35729218a5416ebd74bd1e389f0a568a6ed59b/tests/integrations/server/raft_client.rs#L88

Reply:
Just replace it with `Tikv`.